src/Parser/Mage/Fire/Modules/Features/Combustion.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatNumber } from 'common/format';
+import Analyzer from 'Parser/Core/Analyzer';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+class Combustion extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    spellUsable: SpellUsable,
+  };
+
+  lowPhoenixFlamesCharges = 0;
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.COMBUSTION.id) {
+      return;
+    }
+    const phoenixFlamesCharges = this.spellUsable.chargesAvailable(SPELLS.PHOENIXS_FLAMES.id);
+    if (phoenixFlamesCharges < 2) {
+      this.lowPhoenixFlamesCharges += 1;
+    }
+  }
+
+  get phoenixFlamesChargesSuggestionThresholds() {
+    return {
+      actual: this.lowPhoenixFlamesCharges,
```

Comment:
This would work better as a percentage of Combustion casts where charges were low, because currently it's effectively skewed to be more strict on longer fights (with more Combustion casts)

Reply:
I decided not to do that because you cast like 3-4 Combustion's per fight, so missing one would immediately drop you to 60-70% and that felt awefully punishing. I can change it, it just seemed like it was more punishing than it should be.

Reply:
Just code the suggestions with these numbers in mind. Another problem with the current implementation is that on short fights (eg with only time for 2 combustions), it's literally impossible to get a major issue, even if you fuck them both up (and who starts their 1st combustion with 1 PF charge anway?)

I think you have the right idea that missing even one should be a minor issue. maybe try thresholds of 100%, 65%, 45%? Or if you want to be a little harsher, 100%, 80%, 60%. Basically play in to your knowledge of where the likely breakpoints will be.

Reply:
thats true, i guess it isnt as big of a deal since there is no statistic so the % isnt really visible to the user anyway. Ill change it.